### DISCUSSION TITLE
swagger: flexible editing

### DISCUSSION THREAD

**Discussion by marina-chibizova** [McShelby/hugo-theme-relearn#352] [Q&A] [answered]
Is it possible to change the swagger output so it will show only parts with request params and schemas, without URLs, auth and so on? this parts are not in the open API json, but are still rendered automatically.

**Comment by McShelby** [comment, marked as answer]
Adjusting the swagger output is currently not configurable with shortcode parameters.

But you may could do it in your own CSS. Just take a look into the theme's CSS and search for `rapi-doc` for examples.
